### ISSUE TITLE
Unable to set the `restrictDate` to future

### ISSUE THREAD

**Issue by rahsharm77** [biohzrdmx/jQuery.DatePicker#8]
Hello,

I wanted to thank you for your really great and customizable plugin.

However I wanted to bring to your attention that I am trying to set the restrictDate option to future to only allow future dates but it is not working. Can you show me an example on how to set this option for this plugin?

Thanks,

**Comment by biohzrdmx** [comment]
Hi, the demo page has an example on how to do it: http://biohzrdmx.github.io/jQuery.DatePicker/

Basically, you'll need to pass a `restrictDates` option and set it to `future`, for example:

```javascript
$.datePicker.api.createWidget({
  mode: 'inline',
  container: '.calendar',
  restrictDates: 'future'
});
```

It _should_ work with that, but if it isn't please update the issue with your code and any console output you may get. That would work for the ```datePicker.api.show``` method too.

Regards.

**Comment by rahsharm77** [comment]
Hello,

I am trying to set this options in the defaults method. Something like this:

`$.datePicker = {
        strings: {
            monthsFull: ['January', 'Febraury', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
            monthsShort: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
            daysFull: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
            daysShort: ['Su', 'Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa'],
            messageLocked: 'The day you have just selected is not available'
        },
        defaults: {
            formatDate: function(date) {
                var formatted = dateFormat(new Date(date), 'dddd, mmmm dd, yyyy');
                //console.log(formatted);
                //var formatted = $.datePicker.utils.pad(date.getDate(), 2) + '/' + $.datePicker.utils.pad(date.getMonth() + 1, 2) + '/' + date.getFullYear();
                return formatted;
            },
            parseDate: function(string) {
                string=$.datePicker.defaults.chetanDateFormatter(string);
                var date = new Date();
                var parts = string.match(/(\d{1,2})\/(\d{1,2})\/(\d{4})/);
                if ( parts && parts.length == 4 ) {
                    date = new Date( parts[3], parts[2] - 1, parts[1] );
                }
                return date;
            },
            selectDate: function(date) {
                return true;
            },
            limitCenturies: true,
            closeOnPick: true,
            appendTo: null,
            restrictDates: 'future'
},
//More code
`

Basically my question is that can I set this option in the defaults method or only during the creation time, this option has to be filled in?

Many thanks

**Comment by biohzrdmx** [comment]
Hi, of course you may use the jQuery `extend` function to modify the `defaults` object:

```javascript
$.extend(true, $.datePicker.defaults, {
  restrictDates: 'future'
});
```

Regards.
